Dockerfile

```diff
@@ -57,10 +57,9 @@ VOLUME /var/lib/kvrocks
 
 RUN chown kvrocks:kvrocks /var/lib/kvrocks
 
-COPY ./LICENSE ./NOTICE ./DISCLAIMER ./
+COPY ./docker/run.sh ./LICENSE ./NOTICE ./DISCLAIMER ./
 COPY ./licenses ./licenses
 COPY ./kvrocks.conf /var/lib/kvrocks/
-
 EXPOSE 6666:6666
 
-ENTRYPOINT ["./bin/kvrocks", "-c", "/var/lib/kvrocks/kvrocks.conf", "--dir", "/var/lib/kvrocks", "--pidfile", "/var/run/kvrocks/kvrocks.pid"]
+ENTRYPOINT ["./run.sh"]
```

Comment:
Can we only add the KVROCKS_ARGS for the custom configuration and leave the others as they should(remove the run.sh and docker-compose.yml)?

Reply:
see https://stackoverflow.com/questions/37904682/how-do-i-use-docker-environment-variable-in-entrypoint-array
hard to do this

Reply:
see https://stackoverflow.com/questions/37904682/how-do-i-use-docker-environment-variable-in-entrypoint-array
hard to do this